src/main/java/com/onlinebanking/constant/SecurityConstants.java

```diff
@@ -18,6 +18,7 @@
     public static final String LOGOUT = "/logout";
     public static final String REMEMBER_ME = "remember-me";
     public static final int SECURITY_STRENGTH = 12;
+    public static final String H2_CONSOLE_URL_MAPPING = "/console-h2/**";

```

Comment:
did you make the path /console-h2 intentionally? The default is "/h2-console" 

Just curious